src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/__init__.py

```diff
@@ -152,10 +152,10 @@ def storage_custom_command(self, name, method_name, oauth=False, **kwargs):
             self._register_data_plane_oauth_arguments(command_name)
 
     def storage_custom_command_oauth(self, *args, **kwargs):
-        _merge_new_exception_handler(kwargs, self.get_handler_suppress_403())
+        _merge_new_exception_handler(kwargs, self.get_handler_suppress_some_400())
         self.storage_custom_command(*args, oauth=True, **kwargs)
 
-    def get_handler_suppress_403(self):
+    def get_handler_suppress_some_400(self):
```

Comment:
Instead of putting this here (which will be applied to all data-plane commands), could you scope the exception catching to just the copy command?

Reply:
Can you point me where the copy command will throw the exception? Thanks.